docs-src/clusters/how-to-set-up-visibility-in-a-temporal-cluster.md

```diff
@@ -0,0 +1,38 @@
+---
+id: how-to-set-up-visibility-in-a-temporal-cluster
+title: How to set up Visibility in a Temporal Cluster
+sidebar_label: Visibility store
+description: Visibility storage is set up as a part of your Persistence store to enable listing and filtering details about Worklfow Executions that exist on your Temporal Cluster.
+tags:
+  - operation-guide
+  - filtered-lists
+  - visibility
+---
+
+[Visibility](/concepts/what-is-visibility) storage is set up as a part of your [Persistence store](/concept/what-is-a-temporal-cluster#persistence) to enable listing and filtering details about Worklfow Executions that exist on your Temporal Cluster.
+
+Visibility store is required in a Temporal Cluster setup as it is used by Temporal WebUI and `tctl` to pull Workflow Execution data, and enables features like batch operations on a group of Workflow Executions.
```

Comment:
```suggestion
A Visibility store is required in a Temporal Cluster setup because it is used by Temporal Web UI and `tctl` to pull Workflow Execution data and enables features like batch operations on a group of Workflow Executions.
```